src/style.css

```diff
@@ -1 +1,66 @@
-/* Styles go here */
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  --color-link: #00acdc;
+}
+
+ul li {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+.header {
+  padding-left: 50px;
+  padding-right: 50px;
+  text-transform: uppercase;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.logo {
+  margin-right: 20px;
+  display: flex;
+}
+
+.menu__list {
+  padding: 0;
+  margin: 0;
+  display: flex;
+}
+
+.menu_item {
+  position: relative;
+}
+
+.menu_item a {
+  align-items: center;
+  line-height: 60px;
```

Comment:
don't hardcode height, use padding

Reply:
so, I correct all code, but I dont know how to set height via padding, can you give me a hint in which way to go?

Reply:
box-sizing?

Reply:
please check last commit